traffic_ops/app/lib/API/User.pm

```diff
@@ -277,6 +285,7 @@ sub current {
 			@data, {
 				"id"              => "0",
 				"username"        => $current_username,
+				"tenantId"	  => $self->current_user_tenant(),
```

Comment:
well, ldap will go away with amir's new authentication/authorization model so i'd rather not add another column to the tenant table that will just need to be removed.

i thought you were going to "seed" one tenant with name=root and parent=null (the "root" tenant). and this root tenant was to be the parent of all tenants...

in my opinion, for the short-term, we seed the database with the root tenant and give that tenant to "ldap users". (line 288). this will mean "ldap users" have read-only abilities on ALL resources in the system. (which is what they currently have today).
 
if you set their tenant=undef as you suggested, then they will have read-only role capabilities on nothing. what is the point of that? we will have a few upset users if they log in to the system via ldap and get R/O role and tenant=undef and therefore they can see nothing.